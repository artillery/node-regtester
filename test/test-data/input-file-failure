the contents of this file get mis-capitalized by the capitalizer.
