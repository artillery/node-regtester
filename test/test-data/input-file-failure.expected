THE CONTENTS OF THIS FILE GET MIS-CAPITALIZED by the capitalizer.
